llpc/translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -2527,6 +2527,7 @@ Value *SPIRVToLLVM::transImagePointer(SPIRVValue *spvImagePtr) {
   // generating the code to get the descriptor pointer(s).
   SPIRVWord binding = 0;
   unsigned descriptorSet = 0;
+  ResourceNodeType searchType = ResourceNodeType::Unknown;
```

Comment:
This local variable has a very wide scope. Is that really necessary? I believe that in all cases below, you could just place the required searchType directly as an argument to `getDescPointerAndStride`, without a variable that goes in between.

Reply:
This is my intention to reserve space to insert code in between for unreleased extension. 

i could not do so if placing searchType directly as an argument